package.json

```diff
@@ -8,14 +8,14 @@
   "scripts": {
     "generate-toc": "node ./scripts/generate-toc.mjs",
     "prejson2ts": "npm run build",
-    "json2ts": "./dist/esm/src/cli.js json2ts -i ./example/output/json/components.schemas -o ./example/output/types -c ./example/json-schema-to-typescript-config.json",
+    "json2ts": "./dist/esm/src/cli.js json2ts -i ./example/output/oas2json/components.schemas -o ./example/output/json2ts -c ./example/json-schema-to-typescript-config.json",
     "lint": "eslint .",
     "preoas2json": "npm run build",
-    "oas2json": "./dist/esm/src/cli.js oas2json -i ./example/openapi.yml -o ./example/output/json",
+    "oas2json": "./dist/esm/src/cli.js oas2json -i ./example/openapi_kitchen_sink.yml -o ./example/output/oas2json",
     "preoas2ts": "npm run build",
-    "oas2ts": "./dist/esm/src/cli.js oas2ts -i ./example/openapi.yml -o ./example/output/types -c ./example/json-schema-to-typescript-config.json",
+    "oas2ts": "./dist/esm/src/cli.js oas2ts -i ./example/openapi_kitchen_sink.yml -o ./example/output/oas2ts -c ./example/json-schema-to-typescript-config.json",
     "preoas2tson": "npm run build",
-    "oas2tson": "./dist/esm/src/cli.js oas2tson -i ./example/openapi.yml -o ./example/output/ts",
+    "oas2tson": "./dist/esm/src/cli.js oas2tson -i ./example/openapi_kitchen_sink.yml -o ./example/output/oas2tson",
```

Comment:
We actually don't care if the json files are generated using an OAS: we just need to know that a folder contains json schemas.
Said so, we should rollback from `oas2json` to `json`. The same applies to `json2ts` and `types`.
I would also revert `openapi_kitchen_sink` to plain `openapi`
Changing all of this is not bringing any advantage atm.